Classes/Controller/SearchFEController.php

```diff
@@ -23,6 +23,36 @@ class SearchFEController extends \EWW\Dpf\Controller\AbstractSearchController
 {
 
     /**
+     * action
+     * @var string
+     */
+    protected $action;
+
+    /**
+     * query
+     * @var array
+     */
+    protected $query;
+
+    /**
+     * current page
+     * @var int
+     */
+    protected $currentPage;
```

Comment:
Maybe we should initalize at this point currentPage = 1 as default. This could fix the issue of "previous" link in the result list.
In the old solution "currentPage" was set to 1 as default in searchFEAction

Reply:
@albig, @claussni  

The currentPage is stored in the session data now. So, if no currentPage is explicit used as an argument for the widget, the value will be restored from the session. Therefore you will stay on page 2 (<f:if condition="{pagination.previousPage} > 1">). There is no need for that if-clause at all. See "fix previous link" below.

Reply:
Works on test system.

What's wrong about initializing `$currentPage` with 1, @ruegamer?

Reply:
There is nothing wrong about initializing $currentPage with 1 (I think), but that won't fix this problem

Reply:
Seems, the problem is fixed by https://github.com/kitodo/kitodo-publication/pull/47/commits/8194d8dd0a4800c6c1508981274a62d4c0d227b3